orchagent/portsorch.cpp

```diff
@@ -2303,28 +2307,28 @@ void PortsOrch::doPortTask(Consumer &consumer)
             SWSS_LOG_NOTICE("Deleting Port %s", alias.c_str());
             auto port_id = m_portList[alias].m_port_id;
             auto hif_id = m_portList[alias].m_hif_id;
+            auto bridge_port_oid = m_portList[alias].m_bridge_port_id;
 
-            if (m_portList[alias].has_dependency())
-            {
-                // Port has one or more dependencies, cannot remove
-                SWSS_LOG_WARN("Please remove port dependenc(y/ies):%s",
-                               m_portList[alias].print_dependency().c_str());
+            if (bridge_port_oid != SAI_NULL_OBJECT_ID) {
```

Comment:
Can you put a comment here for what cases we are deferring here?   Also, let's log this event to syslog.